template/nuxt/nuxt.config.ts

```diff
@@ -0,0 +1,182 @@
+
+import { Configuration } from '@nuxt/types'
+
+<%_ if (ui === 'vuetify') { _%>
+import colors from 'vuetify/es5/util/colors'
+<%_ } _%>
+<%_ if (server === 'adonis') { _%>
+  import { resolve } from 'path'
+<%_ } _%>
+
+const config: Configuration = {
+  mode: '<%= mode %>',
+  <%_ if (server === 'adonis') { _%>
+  dev: process.env.NODE_ENV === 'development',
+  srcDir: resolve(__dirname, '..', 'resources'),
+  <%_ } _%>
+  /*
+  ** Headers of the page
+  */
+  head: {
+    <%_ if (ui === 'vuetify') { _%>
+    titleTemplate: '%s - ' + process.env.npm_package_name,
+    <%_ } _%>
+    title: process.env.npm_package_name || '',
+    meta: [
+      { charset: 'utf-8' },
+      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
+      { hid: 'description', name: 'description', content: process.env.npm_package_description || '' }
+    ],
+    link: [
+      <%_ if (ui === 'framevuerk') { _%>
+      {
+        rel: 'stylesheet',
+        href:
+          'https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Material+Icons'
+      },
+      <%_ } _%>
+      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
+    ]
+  },
+  /*
+  ** Customize the progress-bar color
+  */
+  loading: { color: '#fff' },
+  /*
+  ** Global CSS
+  */
+  css: [
+    <%_ if (ui === 'element-ui') { _%>
+    'element-ui/lib/theme-chalk/index.css'
+    <%_ } else if (ui === 'iview') { _%>
+    'iview/dist/styles/iview.css'
+    <%_ } else if (ui === 'ant-design-vue') { _%>
+    'ant-design-vue/dist/antd.css'
+    <%_ } else if (ui === 'tachyons') { _%>
+    'tachyons/css/tachyons.css'
+    <%_ } else if (ui === 'framevuerk') { _%>
+      'framevuerk/dist/framevuerk-nuxt.min.css'
+    <%_ } else if (ui === 'vuesax') { _%>
+    'vuesax/dist/vuesax.css'
+    <%_ } _%>
+  ],
+  /*
+  ** Plugins to load before mounting the App
+  */
+  plugins: [
+    <%_ if (ui === 'element-ui') { _%>
+    '@/plugins/element-ui'
+    <%_ } else if (ui === 'iview') { _%>
+    '@/plugins/iview'
+    <%_ } else if (ui === 'ant-design-vue') { _%>
+    '@/plugins/antd-ui'
+    <%_ } else if (ui === 'framevuerk') { _%>
+    '@/plugins/framevuerk'
+    <%_ } else if (ui === 'vuesax') { _%>
+    '@/plugins/vuesax'
+    <%_ } _%>
+  ],
+  /*
+  ** Nuxt.js dev-modules
+  */
+
```

Comment:
Don't think this carriage return is needed as it's not done for the other comments on top of properties.
```suggestion
```